scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -672,21 +672,22 @@ def test_masked_array_2_sentinel_array():
     # set arbitrary elements to special values
     # (these values might have been considered for use as sentinel values)
     max_float = np.finfo(np.float64).max
-    eps = np.finfo(np.float64).eps
+    max_float2 = np.nextafter(max_float, -np.inf)
+    max_float3 = np.nextafter(max_float2, -np.inf)
```

Comment:
The algorithm for choosing sentinel values changed a bit, so this needed to be updated.